<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStatusForm</class>
 <widget class="QDialog" name="NewStatusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="color" native="true"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="error">
       <property name="text">
        <string> </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
